src/Mod/Arch/ArchIFC.py

```diff
@@ -1,6 +1,8 @@
 import FreeCAD, os, json
 
-"This modules sets up and manages the IFC-related properties, types and attributes of Arch/BIM objects"
+__doc__="""This modules sets up and manages the IFC-related properties, types
```

Comment:
Again, I would not add it to a variable, but instead just change the double quotes, to triple double quotes.

Reply:
Resolved with ed6c261518cd6ee3e5c23e240e91e093a887ca46